install/kubernetes/helm/istio/charts/gateways/templates/clusterrole.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     chart: {{ $.Chart.Name }}-{{ $.Chart.Version }}
     heritage: {{ $.Release.Service }}
     release: {{ $.Release.Name }}
-  name: istio-{{ $key }}-{{ $.Release.Namespace }}
+  name: {{ $key }}-{{ $.Release.Namespace }}
```

Comment:
i saw this during debug a bit too - wasn't sure what was precisely correct-  although I think this is better than istio-istio :)